api_tests/lib_sdk/wallets/ethereum.ts

```diff
@@ -2,15 +2,17 @@ import { BigNumber, EthereumWallet as EthereumWalletSdk } from "comit-sdk";
 import { Asset } from "comit-sdk";
 import { ethers } from "ethers";
 import { BigNumber as BigNumberEthers } from "ethers/utils";
-import { EthereumNodeConfig } from "../../lib/ethereum";
 import { pollUntilMinted, Wallet } from "./index";
-import { HarnessGlobal } from "../../lib/util";
 import { TransactionRequest } from "ethers/providers";
 import * as fs from "fs";
-import { sleep } from "../utils";
+import { HarnessGlobal, sleep } from "../utils";
 
 declare var global: HarnessGlobal;
 
+export interface EthereumNodeConfig {
```

Comment:
That should probably be moved to an `ethereum.ts` file inside the `ledgers` folder rather than moving it in here.